contrib/spec/podman.spec.in

```diff
@@ -392,7 +392,7 @@ popd
 ln -s vendor src
 export GO111MODULE=off
 export GOPATH=$(pwd)/_build:$(pwd):$(pwd):%{gopath}
-export BUILDTAGS="varlink selinux seccomp $(%{hackdir}/hack/btrfs_installed_tag.sh) $(%{hackdir}/hack/btrfs_tag.sh) $(%{hackdir}/hack/libdm_tag.sh) exclude_graphdriver_devicemapper"
+export BUILDTAGS="varlink selinux seccomp systemd $(%{hackdir}/hack/btrfs_installed_tag.sh) $(%{hackdir}/hack/btrfs_tag.sh) $(%{hackdir}/hack/libdm_tag.sh) exclude_graphdriver_devicemapper"
```

Comment:
May be we can use this script https://github.com/containers/libpod/blob/master/hack/systemd_tag.sh ?

Reply:
I refer you to the commit message, in which I explain exactly why I am not doing that.

Reply:
Ah, commit message commit explains it, thanks :-)